userspace/falco/app_actions/load_rules_files.cpp

```diff
@@ -86,12 +86,17 @@ application::run_result application::load_rules_files()
 	}
 
 	falco_logger::log(LOG_DEBUG, "Configured rules filenames:\n");
-	for (auto filename : m_state->config->m_rules_filenames)
+	for (const auto& path : m_state->config->m_rules_filenames)
```

Comment:
I don't think this is wrong, i mean, in log debug, we also print the "loading rules from file X" line, below, while loading rules.
Therefore i thought it would be good to have a point where we print the actually passed-by-user rules paths.

Reply:
I agree Fede, I think this works better